packages/add-changeset/src/addChangesetAction.ts

```diff
@@ -109,7 +109,7 @@ export async function addChangesetAction(context: GitHubActionsContext) {
                 summary,
                 ignore,
                 type,
-                cwd,
+                cwd: cwd === "" ? undefined : cwd,
```

Comment:
Not sure this is actually going to work as expected.

See
https://github.com/isaacs/node-glob/blob/df4598a75a8f5142d1fa2f50e66c686b7527ed7b/common.js#L90

This calls ownProp which call this 
return Object.prototype.hasOwnProperty.call(obj, field)

hasOwnProperty looks like it will report true if a property exists with undefined.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty#using_hasownproperty_to_test_for_an_own_propertys_existence

Reply:
Hopefully they are all the right links

Reply:
I think this might be ok in this instance as previously it would have been an empty string, so the actual `cwd` would have existed previously and still does? Looking at the link to what it does with `cwd` once it checks if the prop exists, it uses `path.resolve(cwd)` which I'm not sure if is going to work. I'll do some testing and may end up reverting this potentially. Thanks for flagging it up.

Reply:
Yep it has an error, I'll fix to specifically use process.cwd if the input isn't provided.

Reply:
That sounds good.

Reply:
Done :)